src/main/java/use_case/suggested_locations/SuggestedLocationOutputData.java

```diff
@@ -0,0 +1,17 @@
+package use_case.suggested_locations;
+
+/**
+ * The output data for the Suggested Location Use Case.
+ */
+public class SuggestedLocationOutputData {
+
+    private final String suggestedLocation;
+
+    public SuggestedLocationOutputData(String suggestedLocation) {
+        this.suggestedLocation = suggestedLocation;
+    }
+
+    public String getSuggestedLocation() {
```

Comment:
this shouldnt be returning a string. getSuggestedLocation should be returning probably a list of either places or strings, that way it can actually be displayed individually on each text field/panel when we open the view. otherwise u just get one really long string as a result?